### ISSUE TITLE
v.1.26+ doesn't support  SSL in python 2.7.6

### ISSUE THREAD

**Issue by yaodingyd** [urllib3/urllib3#2051]
### Subject

Describe the issue here.
It errors out `An HTTP Client raised an unhandled exception: global name 'wrap_socket' is not defined`

### Environment

Describe your environment.
At least, paste here the output of:

```python
import platform
import urllib3

print("OS", platform.platform())
print("Python", platform.python_version())
print("urllib3", urllib3.__version__)
```
Python/2.7.6 botocore/1.19.16 Linux/4.4.0-96-generic

### Steps to Reproduce

A simple and isolated way to reproduce the issue. A code snippet would be great.

I'm calling from latest aws cli which uses urllib3 1.26.1

`aws dynamodb list-tables`

### Expected Behavior

No errors

### Actual Behavior

It errors out `An HTTP Client raised an unhandled exception: global name 'wrap_socket' is not defined`

You may attach logs, packet captures, etc.

**Comment by sethmlarson** [comment]
I'm able to reproduce this issue, will get a fix in and ship 1.26.2 soon.

**Comment by sethmlarson** [comment]
Release is out with the fix.
